python/nav/web/webfront/views.py

```diff
@@ -324,9 +324,14 @@ def preferences(request):
 def qr_code(request):
     """Show qr code linking to current page"""
     url = request.headers.get("referer")
-    qr_code = generate_qr_code_as_string(url=url, caption=url)
+    file_format = webfrontConfig.get("qr_codes", "file_format", fallback="png")
```

Comment:
I don't like the repetition of the fallback value; this config option is read by three different locations in the code, and each location provides a fallback value.  If we want to change the fallback, we now have to change all three instances.

Normally, I would like the default/fallback to be specified in the config parser class itself, but there seems to be a split between the generic "webfront" config parser, and the one provided by `nav.web.auth.ldap` (which provides defaults for only the LDAP section of the config file).

Perhaps we need a WebfrontConfigParser class also in `nav.web`, so we can have common defaults...